tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td

```diff
@@ -2021,7 +2025,8 @@ Our Conv3D implements a form of cross-correlation.
     Confined<I64ArrayAttr, [ArrayMinCount<5>]>:$strides,
     TF_AnyStrAttrOf<["SAME", "VALID"]>:$padding,
     DefaultValuedAttr<TF_AnyStrAttrOf<["NDHWC", "NCDHW"]>, "NDHWC">:$data_format,
-    DefaultValuedAttr<I64ArrayAttr, "{1, 1, 1, 1, 1}">:$dilations
+    DefaultValuedAttr<I64ArrayAttr, "{1, 1, 1, 1, 1}">:$dilations,
+    DefaultValuedAttr<I64ArrayAttr, "{}">:$explicit_paddings
```

Comment:
This information is based on the definition here so we don't have explicit paddings attribute available.
https://github.com/tensorflow/tensorflow/blob/35b14d52e704d316c603dd2599c3c88f329130fe/tensorflow/core/ops/nn_ops.cc#L639

So, we will need to conditionally handle this attribute by fetching attribute based on the name.